src/chat/chat_panel.rs

```diff
@@ -769,7 +745,7 @@ impl ChatPanel {
 
                 self.view(id!(empty_conversation))
                     .label(id!(avatar_label))
-                    .set_text(&get_model_initial_letter(store).unwrap().to_string());
+                    .set_text(&get_model_initial_letter(store).unwrap_or_default().to_string());
```

Comment:
I like the idea of "A" for "Assistant", since it is not looking pretty without any letter as far I tested. In any case, this is a case we should discuss with the design team.